.travis.yml

```diff
@@ -11,7 +11,7 @@ python:
 - '3.4'
 - '3.5'
 install:
-- travis_retry pip install -r requirements-${TRAVIS_PYTHON_VERSION}.txt --allow-external
+- travis_retry pip install -e .
   argparse
```

Comment:
I think argparse is the argument for --allow-external option so should delete both or neither.

Reply:
thanks a lot.